source/iofs/iofs.go

```diff
@@ -31,13 +28,25 @@ func NewDriver(fsys fs.FS, path string) (source.Driver, error) {
 
 // Open is part of source.Driver interface implementation.
 // Open panics when called directly.
-func (d *Driver) Open(url string) (source.Driver, error) {
-	panic("iofs: Driver does not support open with url")
+func (d *driver) Open(url string) (source.Driver, error) {
+	panic("iofs: driver does not support open with url")
```

Comment:
Don't panic and instead return an error: `return nil, errors.New("Open() cannot be called on the iofs passthrough driver")`